community/portworx/templates/autopilot.yaml

```diff
@@ -0,0 +1,123 @@
+{{- if and .Values.csiCloudDrive (eq .Values.csiCloudDrive true)}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: autopilot-config
+  namespace: kube-system
```

Comment:
Need to understand why this offering namespace is kube-system?  Best practices is that software should not be installed into kube-system.  This is just not for this resource but all the new resources that are being introduced.

Reply:
@alweiner Currently, other portworx components are installed in kube-system namespace, So these components also need to be in same namespace.
We are going to add support for custom namespace where user can specify a namespace other than kube-system for existing as well as new PX components. That, we will add it as part of separate PR.